src/test/java/io/xstefank/wildfly/bot/PROpenedTest.java

```diff
@@ -197,6 +211,29 @@ void testDirectoriesMentionsNoHitInDiff() throws IOException {
             });
     }
 
+    @Test
+    void testPullRequestFormatTitleCheckOnOpen() throws IOException {
+        given().github(mocks -> mocks.configFileFromString("wildfly-bot.yml",
+            """
+            wildfly:
+              rules:
+                - title: "Hello"
+                - body: "there"
+                  notify: [7125767235]
+              format:
+                title-check:
+                  pattern: "\\\\[WFLY-\\\\d+\\\\]\\\\s+.*|WFLY-\\\\d+\\\\s+.*"
+                  message: "Wrong content of the title!"
+            """))
+                .when().payloadFromClasspath("/pr-opened.json")
```

Comment:
This will be outdated with JsonHandler PR, so fix it too in that PR please after we merge this.